### PR TITLE
qr-code-component-challenge

### PR DESCRIPTION
**Pull request by akinyiv**

### CODE REVIEW

File: challenges/qr-code-component/index.html [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+            target="_blank"
+            >Optimum BH</a
+          >
+          Internship.
+        </div>
+
+        <div class="mt-3">
+          <a
+            class="text-optimum-blue hover:text-optimum-darkblue"
+            href="https://github.com/optimumBA/frontend_mentor_challenges"
```
Comment (almirsarajcic) [inline challenges/qr-code-component/index.html]: Put your username instead of `optimumBA` so that the link leads to your code.

**Review by almirsarajcic — CHANGES_REQUESTED** [review]
You seem to be on the wrong branch.
This is the correct one: https://github.com/optimumBA/frontend_mentor_challenges/tree/qr_code_component.
Read the instructions again carefully.

Also, the `design` directory is unnecessary.

**Comment by almirsarajcic** [comment]
No need for closing the PR and reopening. You could have just switched the base branch in this PR.

**Comment by akinyiv** [comment]
Hey, yes. I thought of doing that, but then I noticed that my Tailwind is
not responsive, so I am trying to figure out how to fix it.
ᐧ

**Pull request closed without merging**